src/api/app/views/webui/request/beta_show_tabs/_build_status.html.haml

```diff
@@ -7,61 +7,48 @@
-    - if results.present?
-      - previous_repo = nil
-      - results.each do |result|
-        :ruby
-          repository_name = result.repository.tr('.', '_')
-          expanded = repository_expanded?(collapsed_repositories, repository_name, package_name)
-        - if result.repository != previous_repo
-          .d-flex.flex-row.py-1.bg-light.pl-1.pl-sm-2
-            = link_to(word_break(result.repository, 22),
-              package_binaries_path(project: project, package: package, repository: result.repository),
-              title: "Binaries for #{result.repository}")
-            = collapse_link(expanded, package_name, repository_name)
-
-        .collapse{ class: "collapse-#{package_name}-#{repository_name}#{expanded ? ' show' : ''}",
-                   data: { repository: repository_name, main: package_name } }
-          .d-flex.flex-row.flex-wrap.pt-1
-            .repository-state
-              - if result.is_repository_in_db
-                = repository_status_icon(status: result.state)
-              - else
-                %i.fas.fa-clock.text-warning{ title: 'This result is outdated' }
-              %span.ml-1
+  %h5.text-primary.mb-3= package
+  - if results.present?
+    - results_grouped_by_repository = results.group_by(&:repository)
+    - results_grouped_by_repository.each_pair do |repository, results_per_repository|
+      .row.mb-4
+        .col-12.col-sm-3
+          = link_to(word_break(repository, 22),
+            package_binaries_path(project: project, package: package, repository: repository),
+            title: "Binaries for #{repository}")
+        .col-12.col-sm-9
+          - results_per_repository.each do |result|
+            .row.toggle-result-details-parent.mb-1
+              .col-5.col-md-3
                 = result.architecture
-            .build-state.toggle-build-info-parent
-              %i.fa.fa-question-circle.text-info.px-2.pl-lg-1.toggle-build-info{ title: 'Click to keep it open' }
-              = arch_repo_table_cell(result.repository, result.architecture, package, 'code' => result.code, 'details' => result.details)
-            .build-info.mt-1.ml-3.mb-3.mr-3.collapsed
-              .triangle.center
-              .build-info-content
+              .col-5.col-md-2
+                = arch_repo_table_cell(result.repository, result.architecture, package, 'code' => result.code, 'details' => result.details)
+              .col-1.col-md-6
+                - if result.is_repository_in_db
+                  = repository_status_icon(status: result.state)
+                  %span.pl-1.d-none.d-md-inline= repository_info(result.state)
+                - else
+                  %i.fas.fa-clock.text-warning
+                  This result is outdated
+              .col-1.text-right
+                %i.fa.fa-question-circle.text-info.toggle-result-details{ title: 'Click to keep it open' }
+            .row.result-details.bg-light.mb-1.d-none
+              .col.p-3
```

Comment:
This isn't a blocker, but this container and its children have a bit too much padding. We should tone this down. It can happen in another PR.

Reply:
I agree. 

`p-3` is too much:

![Screenshot 2022-10-24 at 11-27-26 Open Build Service](https://user-images.githubusercontent.com/2581944/197495075-c8144da8-37df-4bef-9461-a588b2a9a873.png)

`p-2` is ok for me, `p-1` is not enough:

![Screenshot 2022-10-24 at 11-26-55 Open Build Service](https://user-images.githubusercontent.com/2581944/197495134-b81686f5-10a1-43e6-9781-c7341c7568dc.png)

Reply:
I've reduced the padding/margin between the children too:

![Screenshot 2022-10-24 at 11-51-47 Open Build Service](https://user-images.githubusercontent.com/2581944/197499891-01f2b2c6-c114-4a37-b5e5-463a716f62d4.png)